spec/models/user_spec.rb

```diff
@@ -87,5 +103,20 @@
       user = User.new(email: 'foo@mastodon.space', account: account, password: password)
       expect(user.valid?).to be_truthy
     end
+
+    it 'should not allow a smart user to be created unless they are whitelisted' do
```

Comment:
Instead of "smart user" could you just describe what is being tested? This will become unclear in the future.